watson-developer-cloud/discovery-components
# fix: reset pagination when new query is submitted

#### What do these changes do/fix?
When a new query is submitted, reset the pagination component to show the first page of results.



#### How do you test/verify these changes?
1. Submit a query which returns multiple pages of results
1. Click through to a result page other than 1
1. Submit a new query with multiple pages of results
1. Pagination component should show it's on the 1st page

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
